### PR TITLE
Add Mollie payment links feature

### PR DESCRIPTION
**Pull request by ciungulete**

### CODE REVIEW

File: src/Endpoints/PaymentLinksEndpoint.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Mollie\Api\Endpoints;
+
+use Mollie\Api\Exceptions\ApiException;
+use Mollie\Api\Resources\Payment;
+use Mollie\Api\Resources\PaymentLink;
+use Mollie\Api\Resources\PaymentLinkCollection;
+
+class PaymentLinksEndpoint extends CollectionEndpointAbstract
```
Comment (sandervanhooft) [inline src/Endpoints/PaymentLinksEndpoint.php:10]: Rename to singular `PaymentLinkEndpoint`

File: src/Resources/PaymentLinkCollection.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+namespace Mollie\Api\Resources;
+
+class PaymentLinkCollection extends CursorCollection
+{
+    /**
+     * @return string
+     */
+    public function getCollectionResourceName()
+    {
+        return "paymentLink";
```
Comment (sandervanhooft) [inline src/Resources/PaymentLinkCollection.php:12]: Plural: `paymentLinks`

**Review by sandervanhooft — CHANGES_REQUESTED** [review]

File: src/Resources/PaymentLinkCollection.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+namespace Mollie\Api\Resources;
+
+class PaymentLinkCollection extends CursorCollection
+{
+    /**
+     * @return string
+     */
+    public function getCollectionResourceName()
+    {
+        return "paymentLinks";
```
Comment (sandervanhooft) [inline src/Resources/PaymentLinkCollection.php:12]: I'm checking this with Mollie. The docs state both `paymentLinks` and `payment_links`

**Review by sandervanhooft — CHANGES_REQUESTED** [review]
Any tests yet? 😄

**Comment by skipworkgh** [comment]
Any ETA on this feature?

**Comment by sandervanhooft** [comment]
Can't put a date on it but I have been told it's close. 😀

File: src/Resources/PaymentLinkCollection.php (hunk 3 above)
Comment (sandervanhooft) [reply src/Resources/PaymentLinkCollection.php:12]: According to the API team it's `payment_links`.

**Review by sandervanhooft — APPROVED** [review]

**Pull request merged**